Geometry/VeryForwardGeometry/src/CTPPSPixelSimTopology.cc

```diff
@@ -1,26 +1,31 @@
 #include "Geometry/VeryForwardGeometry/interface/CTPPSPixelSimTopology.h"
```

Comment:
Are the changes in this file necessary? (i.e. doing more than changing the code formatting?)